ql/src/test/results/clientpositive/llap/materialized_view_rewrite_window.q.out

```diff
@@ -293,24 +293,24 @@ STAGE PLANS:
                 Statistics: Num rows: 4 Data size: 480 Basic stats: COMPLETE Column stats: COMPLETE
                 Group By Operator
                   aggregations: max(_col1)
-                  keys: _col2 (type: decimal(12,1))
+                  keys: _col2 (type: decimal(19,4))
```

Comment:
Difference in precision and scale for column. Why has this changed (same below)?